cardinal-components-block/src/main/java/dev/onyxstudios/cca/internal/block/CardinalBlockInternals.java

```diff
@@ -56,6 +56,10 @@ private static synchronized ComponentContainer.Factory<BlockEntity> getBeCompone
             @SuppressWarnings("unchecked") var superclass = (Class<? extends BlockEntity>) entityClass.getSuperclass();
             assert BlockEntity.class.isAssignableFrom(superclass) : "requiresStaticFactory returned false on BlockEntity?";
             factory = /* recursive call */ getBeComponentFactory(superclass);
+
+            // if parent class needs to tick, this one does, too!
+            if(StaticBlockComponentPlugin.INSTANCE.clientTicking.contains(superclass)) StaticBlockComponentPlugin.INSTANCE.clientTicking.add(entityClass);
+            if(StaticBlockComponentPlugin.INSTANCE.serverTicking.contains(superclass)) StaticBlockComponentPlugin.INSTANCE.serverTicking.add(entityClass);
```

Comment:
Also, could you maybe move this to a method in StaticBlockComponentPlugin instead of directly mutating its visible-for-testing "private" fields?

Reply:
I did not change it because `VisibleForTesting`'s semantics say it would otherwise be assumed to be package-private, which would still allow calling it like this just fine.

But sure, will do